pennylane/ops/qubit/parametric_ops.py

```diff
@@ -56,6 +56,9 @@ class RX(Operation):
     grad_method = "A"
     generator = [PauliX, -1 / 2]
 
+    def parameter_frequencies(self):
+        return [(1,)]
```

Comment:
Very very minor, but if a the operator has parameter frequencies that don't depend on `self`, we could ovewrite as a class attribute:
```suggestion
    parameter_frequencies = [(1,)]
```

the main (minor) advantage being that you can query an uninstantiated class

Reply:
wait never mind, I thought `parameter_frequencies` was a property, but it is a method - ignore the above comment!

Reply:
Should it be a property? :thinking: